.github/workflows/publish.yml

```diff
@@ -44,26 +45,12 @@ jobs:
             - run: npm ci
             - run: npm test
 
-            # Use this GitHub Action
             - name: Check package version
               uses: technote-space/package-version-check-action@v1
               with:
                   COMMIT_DISABLED: 1
 
-    publish-npm:
-        needs: [build, get-latest-version]
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v3
-              with:
-                  ref: v${{ needs.get-latest-version.outputs.version }}
-            - uses: actions/setup-node@v3
-              with:
-                  node-version: 22.x
-                  registry-url: https://registry.npmjs.org/
-            - run: git branch
-            - run: npm ci
             - run: npm run build
```

Comment:
the package has a "prepack" rule which already build it. please exclude this line to avoid double run